includes/html/pages/routing/bgp.inc.php

```diff
@@ -313,11 +313,12 @@
         }
 
         unset($sep);
+        $peer_remote = dbFetchRow('SELECT D.device_id FROM `bgpPeers` AS `B`, `devices` AS `D` WHERE `B`.`device_id` = `D`.`device_id` AND B.bgpLocalAddr = ?', [$peer['bgpPeerIdentifier']]);
```

Comment:
```suggestion
        $peer_remote = BgpPeer::where('bgpLocalAddr', $peer['bgpPeerIdentifier'])->value('device_id');
```

Reply:
Super! I've changed the code, it works (of course ;) ) and updated the PR!

(Nice to see the (for me) new Eloquent-way in action -> let's rebase this section later on)